editoast/src/generated_data/error/catenaries.rs

```diff
@@ -59,10 +68,50 @@ pub fn check_catenary_track_ranges(
     infra_errors
 }
 
+/// Checks that catenaries overlapping
+pub fn check_overlapping(infra_cache: &InfraCache, _: &Graph) -> Vec<InfraError> {
+    let mut overlapping_sc = HashSet::new();
+    // Key: (Track, Voltage, Direction)
+    // Value: RangeMap<Range, CatenaryId>
+    let mut range_maps: HashMap<(String, NonBlankString, Direction), RangeMap<u64, String>> =
+        Default::default();
+
+    // Iterate over all the catenaries insure we don't report duplicated errors
+    for catenary in infra_cache.catenaries().values() {
+        let catenary = catenary.unwrap_catenary();
+
+        for track_range in catenary.track_ranges.iter() {
+            let range = (track_range.begin * 100.) as u64..(track_range.end * 100.) as u64;
+            let track_id = &track_range.track.0;
+
+            for direction in get_directions(track_range.applicable_directions) {
```

Comment:
I think you can ignore direction for catenaries since it doesn't make sense for catenaries to be oriented, and the field will be dropped in a future railjson version